docs/install/install-eventing-with-yaml.md

```diff
@@ -0,0 +1,225 @@
+---
+title: "Installing Knative Eventing using YAML files"
+linkTitle: "Install Eventing using YAML"
+weight: 03
+type: "docs"
+showlandingtoc: "false"
+---
+
+This topic describes how to install Knative Eventing.
+You can install the Knative Eventing component by applying YAML files using the `kubectl` CLI.
+
+When installing Knative, you can install the Serving component, Eventing component, or both on your cluster.
+For how to install the Serving component, see [Installing Serving using YAML files](./install-serving-with-yaml.md)
+
+## Prerequisites
+
+Before installation, you must meet the prerequisites.
+See [Knative Prerequisites](./prerequisites.md).
+
+
+## Install the Eventing component
+
+To install the Eventing component:
```

Comment:
seems unnecessary to have this line

Reply:
This text will be expanded as part of issue #3378